SLAM/position_evaluator.py

```diff
@@ -0,0 +1,34 @@
+import SLAM.landmark_matcher
+from SLAM.core import Point
+import math
+
+def evaluate_position(landmark):
+    alpha_A = calculate_alpha(landmark.A.rotation)
+    ZAx = math.sin(alpha_A)*landmark.A.distance
+    ZAy = math.cos(alpha_A)*landmark.A.distance
+    robot_A = Point(None, None, x=landmark.A.x+ZAx, y=landmark.A.y+ZAy)
+
+    alpha_B = calculate_alpha(landmark.B.rotation)
+    ZBx = math.sin(alpha_B)*landmark.B.distance
+    ZBy = math.cos(alpha_B)*landmark.B.distance
+    robot_B = Point(None, None, x=landmark.B.x+ZBx, y=landmark.B.y+ZBy)
+
+    alpha_C = calculate_alpha(landmark.C.rotation)
+    ZCx = math.sin(alpha_C) * landmark.C.distance
+    ZCy = math.cos(alpha_C) * landmark.C.distance
+    robot_C = Point(None, None, x=landmark.B.x+ZCx, y=landmark.C.y+ZCy)
+
+    robot_position = (robot_A+robot_B+robot_C)
+    return robot_position
```

Comment:
You are currently returning the sum of the three estimated positions (`robot_A`, `robot_B`, `robot_C`), but I think you actually want to return the average of the three positions.

Reply:
oh yes of course thanks :)